01_fib/fibonacco_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote("0112358\n")
```

Comment:
This need to print 1 1 2 3 5 8 13

Reply:
Also: it should be newline separated.
Admittedly you have actually provided the correct solution, but it wasn't the lab question :)